Introduction.Rmd

```diff
@@ -125,76 +117,129 @@ The final three chapters describe general best practices not specifically tied t
 
 This is a lot to learn, but don't feel overwhelmed. Start with a minimal subset of useful features (e.g. just an `R/` directory!) and build up over time. To paraphrase the Zen monk Shunryu Suzuki: "Each package is perfect the way it is --- and it can use a little improvement".
 
-## Getting started {#intro-get}
+## devtools, usethis, and you {#intro-usage}
 
-To get started, make sure you have the latest version of R (at least `r paste0(version$major, ".", version$minor)`, which is the version that the code in this book uses), then run the following code to get the packages you'll need:
+> "I am large, I contain multitudes."
+>
+> --- Walt Whitman, Song of Myself
 
-```{r, eval = FALSE}
-install.packages(c("devtools", "roxygen2", "testthat", "knitr"))
+After 7 years of development, devtools had grown into a rather unwieldy package, making maintenance difficult. Version 2.0.0, released in late 2018, marked the [conscious uncoupling](https://www.tidyverse.org/articles/2018/10/devtools-2-0-0/) of devtools, with most functionality moving into seven smaller packages. Through re-exports and wrapper functions, devtools continues to expose all its usual functionality, although it is mostly maintained elsewhere. devtools might provide a wrapper function in order to set more user-friendly defaults, introduce helpful interactive behaviour, or to combine functionality from multiple sub-packages.
+
+Our recommended approach to devtools and its constituent packages:
+
+  * For interactive use, useRs should attach devtools and think of it as the provider of your favorite functions for package development.
+  * For programmatic use, such as in other packages, developeRs should NOT depend on devtools, but should instead access functions via the package that is their primary home.
+    - devtools should rarely appear in the role of `foo` in a qualified call of the form `foo::fcn()`. Instead, `foo` should be the package where `fcn()` is defined.
```

Comment:
You might want to qualify too as being mainly a concern for packages. I don't think it matters much if people use `devtools::use_xyz()` or similar in scripts, though I guess it could be useful to use the direct package for things you are deploying via connect or similar things.

You also have further discussion of this below, so upon further review I think the text is fine as-is.